test/test_spectral_ops.py

```diff
@@ -1125,7 +1125,7 @@ def _test_istft_is_inverse_of_stft(stft_kwargs):
                 for i in range(num_trials):
                     original = torch.randn(*sizes, dtype=dtype, device=device)
                     stft = torch.stft(original, return_complex=True, **stft_kwargs)
-                    inversed = torch.istft(stft, length=original.size(1), **istft_kwargs)
+                    inversed = torch.istft(stft, **istft_kwargs)
```

Comment:
Can you also add test cases that exercise the new code-paths that you added? I.e., the padding and the handling of division by zero?

Reply:
Sure. I have added the test for asserting the paddings are all zero here: https://github.com/pytorch/pytorch/pull/63469/commits/7f219b10bb2d72d25b48f34aa5f1cff8f22170ee